src/styles/add-transaction.css

```diff
@@ -37,19 +37,60 @@
     display: block;
 }
 
-.form-container input {
+.input-block input {
     width: 360px;
     padding: 15px;
     margin: 5px 0 22px 0;
     border: none;
     background: #f1f1f1;
 }
 
-.form-container input:focus {
+.input-block input:focus {
     background-color: #ddd;
     outline: none;
 }
 
+.radios {
+    margin-left: 320px;
+}
+
+.radio-block {
+    display: block;
+    float: left;
+    margin-right: 40px;
+    text-align: center;
+    width: 60px;
+    height: 60px;
+    margin-bottom: 20px;
+    background-color: white;
+    border: 4px solid #ddd;
+    /* border-radius: 30px; */
+}
+
+.radio-block input, label{
+    /* display: block; */
+    /* height: 50px; */
+    cursor: pointer;
+}
+
+.radio-block input[type="radio"] {
+    opacity: 0;
+}
+
+.radio-block label {
+    width: 52px;
+    height: 52px;
+    margin-top: -23px;
```

Comment:
umm doesn't looke too good. Maybe you can achive wanted result in some other way?